Makefile

```diff
@@ -1,15 +1,20 @@
 .PHONY: image test citest update_version
 
 IMAGE_NAME ?= codeclimate/codeclimate-bundler-audit
+TEST_IMAGE_NAME ?= $(IMAGE_NAME)-test
 
 image:
 	docker build --rm -t $(IMAGE_NAME) .
 
-test: image
-	docker run -e PAGER=more --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
+test-image: image
```

Comment:
What is the difference in image size by maintaining a separate Docker image? I'm wondering if the extra complexity of two images/Dockerfiles is worth the gain.

Reply:
The way it's setup in this PR prod image is 19MB/30% smaller than the test image. Prod image is 43.7MB, test image is 62.7MB.

With proposed changes (ruby base image, copy all, bundler-audit cleanup removed) the image ends up clocking in at 74MB. Or 69% larger.